yew/src/html/scope.rs

```diff
@@ -162,7 +162,11 @@ impl<COMP: Component> Scope<COMP> {
         scheduler().push_comp(ComponentRunnableType::Destroy, Box::new(destroy));
     }
 
-    /// Send a message to the component
+    /// Send a message to the component.
+    ///
+    /// Please be aware that currently this method
+    /// immediately/synchronously schedules a call to the
+    /// [Component](Component) interface. Check out
```

Comment:
nit: can you cleanup "Check out"

Reply:
will do, I got halfway through mentioning the LinkFuture when I realised it hadn't been released yet, and forgot to remove that :sweat_smile:

Reply:
🤣  no worries!